Expanding the SSO FAQ (#2455)
buildkite/docs: pages/integrations/sso.md

### Why do I get the error "we couldn't find an account with that email address" when logging in?

This is likely caused by trying to log in from the wrong place. You need to log in from https://buildkite.com/sso and follow the link from the email you receive. If the issue persists, please reach out to support@buildkite.com for assistance.

> ### Will setting the session duration affect all current sessions or only the new sessions?
> When you [update the session duration](https://buildkite.com/docs/apis/graphql/cookbooks/organizations#update-the-default-sso-provider-session-duration), it affects both the new and the old SSO sessions.

Reviewer: Formatting, link syntax, and an optional nitpick for brevity (removing "the"):

```suggestion
### Will setting the session duration affect all current sessions or only the new sessions?

When you [update the session duration](/docs/apis/graphql/cookbooks/organizations#update-the-default-sso-provider-session-duration), it affects both new and old SSO sessions.
```

> ### Which even is considered to be the start of an SSO session?
> 
> The start of an SSO session is individually measured per user from the moment that user signs in via SSO.

Reviewer: I found this a bit confusing but might have simplified too much:

```suggestion
### When is an SSO session considered to start?

An SSO session starts for a user from the moment they sign in using SSO.
```
